ext/mimemagic/Rakefile

```diff
@@ -24,7 +24,7 @@ end
 desc "Build a file pointing at the database"
 task :default do
   mime_database_path = locate_mime_database
-  target_dir = "#{ENV.fetch("RUBYARCHDIR", "../lib")}/mimemagic"
+  target_dir = File.join(RbConfig::CONFIG['rubyarchdir'] || "../lib", "mimemagic")
```

Comment:
On second thought, I'm not sure this change is correct. Fallback is not needed, because `RbConfig::CONFIG['rubyarchdir']` will always be present. While in old code `RUBYARCHDIR` would be defined... almost never? I don't know, really. What was the purpose of this line originally?